Gruntfile.js

```diff
@@ -68,6 +74,15 @@ module.exports = function (grunt) {
             }
         },
 
+        file_append: {
+            addSourceURL: {
+                files: [{
```

Comment:
Make sure you have you linter enabled in JavaScript too.
Missing white space between `[ {` and `} ]`